package.json

```diff
@@ -230,12 +230,12 @@
 		"postinstall": "node ./node_modules/vscode/bin/install"
 	},
 	"dependencies": {
-		"vscode-debugadapter": "^1.11.0",
-		"vscode-debugprotocol": "^1.11.0",
+		"vscode-debugadapter": "^1.12.0",
```

Comment:
I don't, but I was able to run w/ these versions on the stable channel. We're not really making use of the newer APIs; I see they were published to npm ~2 weeks ago.